### PR TITLE
Automating the release process

### PR DESCRIPTION
**Pull request by robertpatrick**
Details of the automated process can be found in the ReleaseProcess.md file at the root of the project.

### CODE REVIEW

File: ReleaseProcess.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      <passphrase>store plain text or encrypted token here</passphrase>
+    </server>
+  </servers>
+```
+
+## Software Release Process
+The best practice is to write the release notes that will be published to GitHub prior to starting the steps below.
+
+1. Set (and export) the environment variable `WLST_DIR` to `<WLS-install-dir>/oracle_common/common/bin`, replacing `<WLS-install-dir>` with the full path to the WLS 12.2.1.x installation directory.
+2. In the weblogic-deploy-tooling project directory, create a file called release.properties with content similar to the example shown below.  Note that the example is configured to cut the 1.9.11 release.
```
Comment (ddsharpe) [inline ReleaseProcess.md:23]: add reverse tick emphasis to filename, `release.properties`

Comment (robertpatrick) [reply ReleaseProcess.md]: done

**Review by ddsharpe — APPROVED** [review]

File: ReleaseProcess.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+releaseVersion=1.9.11
+developmentVersion=1.9.12-SNAPSHOT
+```
+
+3. In the weblogic-deploy-tooling project directory, run the `mvn -B -P release release:prepare release:perform` command.  If your SSH private key has a passphrase, watch the build closely since it will prompt for your passphrase multiple times.  Failure to enter it in a timely manner may result in a failure.
+4. If the build fails, run the `mvn -B -P release release:rollback` command to undo it and start over from Step 2 after correcting the issue.
+5. Once the software has been released, move on to the GitHub Release Creation Process.
+
+## GitHub Release Process
+Note that this process relies on the WDT installers being in your local Maven repository.  As such, it is critical for the same user to run these steps on the same machine as the steps from the previous section!
```
Comment (jshum2479) [inline ReleaseProcess.md:36]: installers? or installer?

**Review by jshum2479 — APPROVED** [review]

File: ReleaseProcess.md (hunk 2 above)
Comment (robertpatrick) [reply ReleaseProcess.md:36]: I added a second installer with a tar.gz extension

**Pull request merged**